src/main/kotlin/xyz/atrius/waystones/data/config/Parsers.kt

```diff
@@ -59,6 +60,12 @@ object BooleanParser : ArgumentParser<Boolean> {
         if (input?.toLowerCase() in arrayOf("true", "false")) input.toBoolean() else null
 }
 
+object LocaleParser : ArgumentParser<Locale> {
+    override fun parse(input: String?): Locale? = input?.let { Locale.forLanguageTag(input) }
```

Comment:
We should probably enforce some form of format before setting the value as mentioned in another comment

Reply:
`Locale.forLanguageTag()` accepts any `String`, there will be no error thrown for sure.

If the Locale is something that `MessageFormat` does not recognize, it will probably treat it in a generic way.

Reply:
Any suggestions for what we should this?

Reply:
Some documentation about this behaviour should be enough.

Reply:
Probably should add documentation for it to the README file then.